src/axom/slic/core/Logger.hpp

```diff
@@ -66,18 +66,36 @@ class Logger
   void setLoggingMsgLevel(message::Level level);
 
   /*!
-   * \brief Aborts and flushes on warning or error if corresponding AbortOnError
-   *  or AbortOnWarning is set to true
-   * \collective
+   * \brief Gets the abort flag for the given level.
```

Comment:
white space between the brief statement and param statements would increase readability. Same for white space between param statements and return statements.

This comment applies to all methods in these files.